test/src/test/java/org/corfudb/runtime/collections/UFOStoreTestImpl.java

```diff
@@ -0,0 +1,145 @@
+package org.corfudb.runtime.collections;
+
+import com.google.protobuf.Message;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.CorfuStoreMetadata;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.lang.reflect.InvocationTargetException;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * UFOStore is a thin layer over CorfuStore that provides certain metadata management
+ * that carry business logic specific to verticals.
+ *
+ * Created by hisundar on 2020-09-16
+ */
+class UFOStoreTestImpl {
```

Comment:
Can you rename it to something else?  Because we also have a UFOStoreTest which is a test class and not related to this impl.

Reply:
done.